### PR TITLE
ref: code reorg rename

### PR DESCRIPTION
**Pull request by nksazonov**

### CODE REVIEW

File: contracts/interfaces/IClearingTypes.sol [hunk 1]
```diff
@@ -6,9 +6,9 @@ import '@statechannels/nitro-protocol/contracts/interfaces/INitroTypes.sol';
 /**
  * @notice Interface
  */
-interface IMarginApp {
+interface IClearingTypes {
```
Comment (mod) [inline contracts/interfaces/IClearingTypes.sol:9]: Maybe ISettlement ?

Comment (mod) [reply contracts/interfaces/IClearingTypes.sol:9]: and IMargin so we separate the two types

File: contracts/interfaces/IClearingTypes.sol [hunk 2]
```diff
@@ -6,9 +6,9 @@ import '@statechannels/nitro-protocol/contracts/interfaces/INitroTypes.sol';
...
 	enum MarginIndices {
-		Leader,
+		Initiator,
 		Follower
```
Comment (mod) [inline contracts/interfaces/IClearingTypes.sol:12]: Client, Server

File: contracts/interfaces/IClearingTypes.sol (hunk 1 above)
Comment (nksazonov) [reply contracts/interfaces/IClearingTypes.sol]: 17fa384

**Pull request merged**